CMakeLists.txt

```diff
@@ -87,6 +87,8 @@ include(CMakeFuncs)
 
 set(openssl_min 1.0.2)
 
+project(rippled)
```

Comment:
I'd move this _all_ the way up, to the line just below the min cmake version. It works as-is, but unless it's higher someone is going to be tempted to put code above the `project` and we've seen how that can break things.